### PR TITLE
Remove volunteer date time

### LINKED ISSUE #92: Remove volunteer date and time parameters

### PR DESCRIPTION
**Pull request by Fluffykan**
Removes the volunteer class available date, start time and end time parameters since they are no longer in use.

Closes #92

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ExportCommand.java
Comment (JovenSoh) [inline src/main/java/seedu/address/logic/commands/ExportCommand.java:1]: Should we add in the events the volunteers are involved in?

File: src/main/java/seedu/address/model/volunteer/Volunteer.java
Comment (JovenSoh) [inline src/main/java/seedu/address/model/volunteer/Volunteer.java:1]: Should we take away the `availableDate=` here too?

**Review by JovenSoh — CHANGES_REQUESTED** [review]
Some small changes changes to be made

File: src/main/java/seedu/address/model/volunteer/Volunteer.java
Comment (Fluffykan) [reply src/main/java/seedu/address/model/volunteer/Volunteer.java:1]: yes we should, mb, thks for the catch.

File: src/main/java/seedu/address/logic/commands/ExportCommand.java
Comment (Fluffykan) [reply src/main/java/seedu/address/logic/commands/ExportCommand.java:1]: i think ill handle this in a separate PR, cos by the looks of it, we're not deleting both date and time as of now. thanks for the reminder 👍

**Comment by Fluffykan** [comment]
we are not removing the 2 parameters anymore

**Pull request closed without merging**